app/models/manageiq/providers/kubevirt/infra_manager/refresh_worker/runner.rb

```diff
@@ -53,13 +53,17 @@ def before_exit(_message, _exit_code)
 
   private
 
+  def provider_class
+    ManageIQ::Providers::Kubevirt
+  end
```

Comment:
For a follow-up, if we have the `@ems` from the base class this can be `@ems.class.module_parent` and then you don't need the overrides in the openshift plugin https://github.com/ManageIQ/manageiq-providers-openshift/pull/264/files#diff-58345ef2b8d148162c00a99ed61d27627540d638f0d2b9a56f9c0472e9a26524R6-R8